go/libraries/doltcore/schema/typeinfo/varbinary.go

```diff
@@ -171,11 +142,11 @@ func (ti *varBinaryType) IsValid(v types.Value) bool {
 
 // NomsKind implements TypeInfo interface.
 func (ti *varBinaryType) NomsKind() types.NomsKind {
-	return types.StringKind
+	return types.ChunkedStringKind
```

Comment:
I thought we had decided that the noms kind for these types depends on the length of the sql type. For LongBlob this makes sense, for other types not as much.

Reply:
So `BINARY/VARBINARY` use `inlineBlobType`, where as all of the `BLOB` types use `varBinaryType`, or `InlineBlob` and `ChunkedString` respectively.

Although the smaller `BLOB` types could technically be inlined as well, this mimics MySQL's implementation (where all `BLOB` types are stored out of band), and really someone should just use a `VARBINARY` over any `TINYBLOB`.